### PR TITLE
setup prettier and eslint

### LINKED ISSUE #220: Add prettier for code formatting
Add prettier so code should be formatted
More info: https://prettier.io/docs/en/install.html

### LINKED ISSUE #223: Setting up ESLint in the project
## Is your feature request related to a problem? Please describe.

As the project grows bigger, it is important that the code is linted properly to ensure the consistency and quality of the code
To achieve this, we should set up ESLint in the project.

### PR DESCRIPTION
**Pull request by gnikita432**
#### Description
Issue FIXES #220
FIXES #223 

#### Describe the changes you've made

Added prettier and eslint setup also formatted all the code with prettier

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)

NA
#### Checklist

- [ ] I have added my name in the contributors list at the end of README.md file.
- [x] My code follows the code style of this project.
- x ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [x] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)GSSOC

### CODE REVIEW

**Comment by gnikita432** [comment]
hey @K-Kumar-01 @khushishikhu @Aaishpra  I have made a new pull request for adding prettier and eslint. can you please review this PR because as new PRs will get merge it will create conflicts again and again.

File: periodo/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": ["airbnb", "prettier", "plugin:node/recommended"],
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": "error",
+    "no-unused-vars": "warn",
```
Comment (K-Kumar-01) [inline periodo/.eslintrc.json:6]: make `no-unused-vars` to `error`

File: periodo/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "no-console": "off",
```
Comment (K-Kumar-01) [inline periodo/.eslintrc.json:7]: I think console can be made to `warn`

**Comment by gnikita432** [comment]
yes, I have made the changes.

**Comment by K-Kumar-01** [comment]
@gnikita432 
Please ensure that you follow the commit guidelines mentioned in the `CONTRIBUTING.md` in the repo.

**Comment by Aaishpra** [comment]
Hi,@gnikita432 @K-Kumar-01 is talking about [this](https://github.com/Feminine-Divine/periodo/blob/master/.github/CONTRIBUTING.md#commit-conventions) commit message guidelines Periodo is using, commit messages are an important part of open-source projects when you are contributing. So according to the guidelines a good commit message for this issue will be
`feat: setup eslint and Prettier - #220 #223`

**Review by Aaishpra — APPROVED** [review]
It looks alright to me, @K-Kumar-01 can review this again and merge it

**Comment by gnikita432** [comment]
Okay, Sorry for the inconvenience I will change my commit message. 
Also After solving a merge conflict it is showing all checks have failed, Is there any issue with the PR?

**Comment by Aaishpra** [comment]
Don't worry about the checks these are just deployment checks from vercel

**Comment by K-Kumar-01** [comment]
LGTM 🎉

**Pull request merged**